python/ray/data/read_api.py

```diff
@@ -525,7 +528,7 @@ def read_bigquery(
 
     Args:
         project_id: The name of the associated Google Cloud Project that hosts the dataset to read.
-            For more information, see `Creating and Managing Projects <https://cloud.google.com/resource-manager/docs/creating-managing-projects>`. # noqa: E501
```

Comment:
Move `noqa` to end of docstring?

https://github.com/ray-project/ray/blob/8ecf2f5be0dfda4e2925da2ee1f08d7771e72e5e/python/ray/data/dataset.py#L357-L357